languages/csharp/exercises/concept/chars/.docs/after.md

```diff
@@ -0,0 +1,78 @@
+- [Chars documentation][chars-docs]: reference documentation for `char`.
+- [Chars tutorial][chars-tutorial]: basic tutorial on how to work with chars.
```

Comment:
We usually don't have links to start the document with. There is no hard rule there though, but I think I prefer having a short introduction text first.